File: ProyectoBilletera/Models/Cliente.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

#nullable disable

namespace WepAppClip.Models
{
    public partial class Cliente
    {
        public Cliente()
        {
            Cuenta = new HashSet<Cuentum>();
        }

        public int IdCliente { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public int? Estado { get; set; }
        public int? IdDireccion { get; set; }
        public string NroTelefono { get; set; }
        public string NroDni { get; set; }
        public byte[] FrontalDni { get; set; }
        public byte[] TraseraDni { get; set; }
        public string Email { get; set; }
        [JsonIgnore]
        public string Password { get; set; }

        public virtual Direccion IdDireccionNavigation { get; set; }
        public virtual ICollection<Cuentum> Cuenta { get; set; }
    }
}

File: billeteraClip/ClientApp/src/app/models/cliente.ts
export interface Cliente {
    nombre: string,
    apellido: string,
    estado: number,
    idDireccion: number,
    nroTelefono: string,
    nroDni: string,
    frontalDni: any,
    traseraDni: any,
    email: string,
    password: string
}

export interface ClienteViewModel {
  nombre: string,
  apellido: string
  nroTelefono: string,
  nroDni: string,
  frontalDni: any,
  traseraDni: any,
  email: string,
  password: string,
  direccion: string,
  alturaCalle: string,
  ciudad: string,
  provincia: string

}
